components/breadcrumb/demo/basic.md

```diff
@@ -1,10 +1,18 @@
 ---
 order: 0
-title: 基本
+title:   
+  zh-CN: 基本
+  en-US: Basic
 ---
 
+## zh-CN
+
 最简单的用法。
 
+## en-US
+
+The simplest use
+
 ````jsx
 import { Breadcrumb } from 'antd';
 
```

Comment:
I suspect you should translate the Chinese text in the example blocks of code in here as well.